test/integration/targets/lookup_mongodb/tasks/main.yml

```
@@ -0,0 +1,36 @@
+# test code for the mongodb lookup module
+# (c) 2018, Marcos Diez <marcos AT unitron.com.br>
+
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+
+- include: test_lookup_mongo.yml
+  when: ansible_distribution == 'Ubuntu' and ansible_distribution_release == "xenial"
```

Is check for Xenial really necessery? We have at least newer LTS version for Ubuntu. I just tried that with 18.04 and it works.
```suggestion
  when: ansible_distribution == 'Ubuntu'
```

---

No. It's not and it has been removed. When this was done, bionic (18.04) did not exist, xenial (16.04) was new and there were still ansible tests for 14.04, which either did not contain mongodb or pymongo, which is why this lines was there. Thanks for noticing.

---

No. It's not and it has been removed. When this was done, bionic (18.04) did not exist, xenial (16.04) was new and there were still ansible tests for 14.04, which either did not contain mongodb or pymongo, which is why this lines was there. Thanks for noticing.